bloop.json

```diff
@@ -3,14 +3,11 @@
   "url": "https://github.com/scalacenter/bloop/releases/download/v1.4.1/bloop-coursier.json",
   "hash": "sha256:2e6a873183e5e22712913bfdab1331d0a7792167c369fee5be0c83e27572fe12",
   "depends": "coursier",
-  "bin": "bloop",
-  "env_add_path": "$dir",
+  "bin": "bloop.bat",
   "env_set": {
-    "BLOOP_HOME": "$dir",
-    "BLOOP_IN_SCOOP": "true"
+    "BLOOP_HOME": "$dir"
   },
   "installer": {
-    "script": "coursier install --install-dir $dir --default-channels=false --channel $dir bloop"
+    "script": "coursier install --install-dir $dir bloop"
```

Comment:
Should we add `--prefer-prebuilt true` here to avoid coursier trying to compile the application to GraalVM native?

Reply:
I tried installing bloop with scoop using this version of line 11.  It appears to install properly but can someone double-check?  
```
    "script": "coursier install --install-dir $dir --default-channels=false --channel io.get-coursier:apps bloop"
```
I'm still learning, but my (fuzzy) understanding is `io.get-coursier:apps` (ref: [coursier docs](https://get-coursier.io/docs/cli-install.html#jar-based-channels)) is a "Maven coordinate" and it redirects to bloop's most recent jar.  Does that sound right?